firebase-inappmessaging/src/test/java/com/google/firebase/inappmessaging/internal/InAppMessageStreamManagerTest.java

```diff
@@ -198,6 +198,8 @@ public void setup() {
             abtIntegrationHelper);
```

Comment:
PLs consider a new test that checks is the completable was subscribed to

Reply:
this is tested (somewhat indirectly) by our flowable test